src/main.rs

```diff
@@ -235,10 +235,20 @@ fn show_paths() {
                 .unwrap_or_else(|_| String::from("[Invalid]"))
         },
     );
-    println!("Config dir:  {}", config_dir);
-    println!("Config path: {}", config_path);
-    println!("Cache dir:   {}", cache_dir);
-    println!("Pages dir:   {}", pages_dir);
+
+    // This first unwrap will never panic because RawDirectoriesConfig implements Default. The only
+    // reason this is Option is if get_app_root() fails.
+    let custom_pages_dir = custom_pages_dir
+        .unwrap()
```

Comment:
This is guarenteed to be safe. Kinda counter-intuitive. It is only Option because of how Default for the RawDirectoryConfig is implemented with get_app_root. But since there is a default it is safe to unwrap.